examples/experimental/skip_test/weave_walkthrough.ipynb

```diff
@@ -0,0 +1,415 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f8febdb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "weave.use_frontend_devmode()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71ffc2ae",
+   "metadata": {},
+   "source": [
+    "## Intro\n",
```

Comment:
This is a nice general overview of Weave, maybe add one sentence explaining what will be covered in this notebook specifically?

Reply:
added: This notebook is an evolving introduction to Weave—it walks through the core concepts and existing patterns for composing, manipulating, and extending a compute graph of typed objects.